Fix NU5128 on pack

[dotnet pack](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-pack) on `F0.Analyzers` reports _warning NU5128_.

```
warning NU5128: Some target frameworks declared in the dependencies group of the nuspec and the lib/ref folder do not have exact matches in the other location. Consult the list of actions below:
warning NU5128: - Add lib or ref assemblies for the netstandard1.3 target framework
```

Currently, warnings are suppressed with `<NoPackageAnalysis>true</NoPackageAnalysis>` in _Configuration=Release_.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,8 @@
 CHANGELOG
 
 ## vNext
+### NuGet package
+- Suppressed package dependencies from generated NuGet package.
 
 ## v0.3.0 (2020-05-27)
 ### Analyzers

--- a/source/production/F0.Analyzers/F0.Analyzers.csproj
+++ b/source/production/F0.Analyzers/F0.Analyzers.csproj
@@ -8,6 +8,8 @@
 
   <PropertyGroup>
     <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IsPackable>true</IsPackable>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
     <DevelopmentDependency>true</DevelopmentDependency>
   </PropertyGroup>
 
@@ -28,15 +30,7 @@
     <RepositoryUrl>https://github.com/Flash0ver/F0.Analyzers.git</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <PackageTags>f[0], f0, analyzers, roslyn, csharp</PackageTags>
-    <PackageReleaseNotes>Added goto analyzer (F00001) which reports diagnostics for goto statements in user code.</PackageReleaseNotes>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
-    <NoPackageAnalysis>false</NoPackageAnalysis>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
-    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <PackageReleaseNotes>Skipped all the dependencies from generated nupkg file.</PackageReleaseNotes>
   </PropertyGroup>
 
   <ItemGroup>

